xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -349,6 +353,9 @@ bool CRenderManager::Configure()
       return false;
   }
 
+  // configure the renderer to crop if necessary
+  m_pRenderer->ConfigureCropping(m_cropBottom, m_cropTop, m_cropLeft, m_cropRight);
```

Comment:
Please add the cropping parameter to configure.

> I also wasn't quite sure how to test my changes

our CI system will do. I can help too.

btw: if you update the PR without adding some comment, we don't get notification. I almost missed your last changes.